.github/workflows/build.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         cd obs-studio/build
 
         export QTDIR=$(brew --prefix qt5)
+        export MACOSX_DEPLOYMENT_TARGET=10.14
```

Comment:
I _think_ we can set this as an environment variable for the entire job like this.

```
jobs:
  build:
    env:
      MACOSX_DEPLOYMENT_TARGET: 10.14
    runs-on: macos-latest
```

Reply:
Right. Done. Thanks.